frontend/__tests__/common/SimpleModal.test.tsx

```diff
@@ -0,0 +1,77 @@
+import "@testing-library/jest-dom"
+import { render, screen } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import SimpleModal, {
+  type SimpleModalProps,
+} from "../../src/components/Common/SimpleModal"
+
+describe("SimpleModal", () => {
```

Comment:
nit: Call this "Confirmation Modal"

Reply:
I don't love the name simple modal but its purposefully named such that there are no assumptions about its use. How about ModalWrapper?

Reply:
ContentModal?  MessageModal?

Reply:
Changed to ContentModal